.version-support-mysql.json

```diff
@@ -1,5 +1,6 @@
 {
 	"6-8": [
+		"9.1",
 		"9.0",
```

Comment:
Should 9.0 be removed here since we are only supporting the current Innovation Release?

Reply:
Do we only support the latest innovation release? I think that's reasonable, but we need to post a Make Core proposal around that before we can consider it the practice. For the 8.x releases, we included all innovation releases in this support file until the next LTS was tagged. 

This aims to use that same practice, but only test against the _latest_ innovation release.